bool start_chat
---
bool chat_success
